### ISSUE TITLE
end-2-end failures 

### ISSUE THREAD

**Issue by maxandersen** [openshiftio/openshift.io#1512]
Tracking issues for known issues blocking end-2-end work

- [ ] Forge fails to create quickstarts #1503 
- [ ] Che init issues #1304 
- [ ] Jenkins pipeline not created reliably #1194

**Comment by maxandersen** [comment]
@ldimaggi please help keep this task list above uptodate on what blocks end-2-end and tick them off when you are unblocked.

**Comment by ibuziuk** [comment]
e2e tests are failing on both prod & prod-preview since Jan 24th due to https://github.com/openshiftio/openshift.io/issues/2034. Shouldn't this issue be marked as SEV1 ?

**Comment by ldimaggi** [comment]
Sev1 is for production only, not prod-preview,

This is a bad bug, but, the user is not 100% blocked as they can promote from either the Jenkins build log or from OSO.

But - yes - we should fix this ASAP...

**Comment by ldimaggi** [comment]
This issue lists some very old issues - if we want to have a parent JIRA for all new failures, I'd recommend opening a new issue. Thx!
